oap-server/server-starter/src/main/resources/meter-analyzer-config/golang.yaml

```diff
@@ -0,0 +1,34 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+expSuffix: instance(['service'], ['instance'], Layer.GENERAL)
+metricPrefix: meter
+metricsRules:
+  - name: instance_golang_heap
+    exp: instance_golang_heap
+  - name: instance_golang_stack
+    exp: instance_golang_stack
+  - name: instance_golang_gc_time
+    exp: instance_golang_gc_time
```

Comment:
Is this a gauge?

Reply:
instance_golang_stack represents the bytes in stack spans right now
instance_golang_gc_time represents  the most recent gc pause time
those are MeterSingleValue?

Reply:
My point is about GC time. How a metric could be called recent? 
What is the time duration about the recent?

Reply:
> 
time duration is decided by user input, if user input is 15 seconds, than every 15 seconds will collect the latest gc's puase time using

Reply:
Then, what is your set? As the GC time is splitted, you should use sum function to merge, and use sum downsampling for hour and day level metric aggregation.

Reply:
The default downsampling is an avg, check hour level GC time you set now, you would see a hour GC time is close to a minute GC time.

Reply:
OK, Should I change to "instance_golang_gc_time.downsampling(SUM)" ?

Reply:
OK, thanks, I will test this method